iina/QuickSettingViewController.swift

```diff
@@ -505,6 +509,9 @@ class QuickSettingViewController: NSViewController, NSTableViewDataSource, NSTab
 
   @IBAction func customAudioDelayEditFinishedAction(_ sender: NSTextField) {
     let value = sender.doubleValue
+    if value == 0 {
```

Comment:
At first glance it looks like this code doesn't do anything, so I'd prefer `sender.stringValue.isEmpty`, since it makes the meaning explicit.